### DISCUSSION TITLE
在启动时合并消息相关的代码出现问题

### DISCUSSION THREAD

**Discussion by strll** [simple-robot/Discussions#42] [Q&A]
我在使用合并消息的时候出现了 下面这个错误 
无法访问love.forte.simbot.event.ChatRoomMessageEvent
找不到love.forte.simbot.event.ChatRoomMessageEvent的类文件

即使我重新下载依赖并且回退到之前已经上线的版本也还是会出现这个错误

**Comment by strll** [comment]
在依赖中我发现这个是ChatroomMessageEvent  报错的是找不到ChatRoomMessageEvent 我猜测应该是这个R的大小写区别导致这个问题 但是我不知道该怎么解决

**Reply by ForteScarlet** [reply]
大概率为版本问题，请更新版本后再做尝试。如果仍有问题，反馈问题至[issues](https://github.com/simple-robot/simpler-robot/issues)而非 _讨论区_ 。

simbot3中从未有过 `ChatroomMessageEvent`，一直都是 `ChatRoomMessageEvent`，参考[MessageEvents.kt](https://github.com/simple-robot/simpler-robot/blob/v3-main/simbot-apis/simbot-api/src/main/kotlin/love/forte/simbot/event/MessageEvents.kt#L160) 及其 [历史记录](https://github.com/simple-robot/simpler-robot/commits/v3-main/simbot-apis/simbot-api/src/main/kotlin/love/forte/simbot/event/MessageEvents.kt)
